docker-compose.yml

```diff
@@ -20,12 +20,12 @@ services:
     container_name: app_db
     hostname: app_db
     image: library/postgres:14.1
-    environment:
-      - POSTGRES_USER=app
-      - POSTGRES_PASSWORD=app
-      - POSTGRES_DB=app
+    env_file:
+      - .env
```

Comment:
👍

Reply:
@zhanymkanov about the variable `DATABASE_URL`. If we have to store all **Postgres** variables, maybe we should generate `DATABASE_URL` with PostgresDsn from Pydantic?

Otherwise, we should write variables in POSTGRES variables + `DATABASE_URL`

Reply:
yeah, that would be nice, but it will fit better another PR